### ISSUE TITLE
new version - error with source maps for build

### ISSUE THREAD

**Issue by hkollmann** [qooxdoo/qooxdoo-compiler#41]
```Versions: qooxdoo-cli v0.1.32, qooxdoo-compiler v0.1.27```

Create a new project with ```qx create --verbose --type=desktop test```
run ```qx compile --target=build```

this is the result:
```
Error: original.line and original.column are not numbers -- you probably meant to omit the original mapping entirely and only map the generated position. If so, pass null for the original mapping instead of an object with empty or null values.
    at SourceMapGenerator_validateMapping [as _validateMapping] (C:\_vza\@repos\github\qx-cli\node_modules\source-map\lib\source-map-generator.js:267:15)
    at SourceMapGenerator_addMapping [as addMapping] (C:\_vza\@repos\github\qx-cli\node_modules\source-map\lib\source-map-generator.js:101:12)
    at C:\_vza\@repos\github\qx-cli\node_modules\qxcompiler\lib\qxcompiler\targets\BuildTarget.js:210:46
    at Array.forEach (<anonymous>)
    at BasicSourceMapConsumer.SourceMapConsumer_eachMapping [as eachMapping] (C:\_vza\@repos\github\qx-cli\node_modules\source-map\lib\source-map-consumer.js:155:14)
    at fs.readFileAsync.then (C:\_vza\@repos\github\qx-cli\node_modules\qxcompiler\lib\qxcompiler\targets\BuildTarget.js:198:31)
    at <anonymous>
```

**Comment by johnspackman** [comment]
Fixed in qooxdoo-cli v0.1.33, qooxdoo-compiler v0.1.28

**Comment by hkollmann** [comment]
Great
